core/src/main/java/hudson/model/AdministrativeMonitor.java

```diff
@@ -131,7 +131,7 @@ public void disable(boolean value) throws IOException {
      * he wants to ignore.
      */
     public boolean isEnabled() {
-        return !((AbstractCIBase)Jenkins.getInstance()).disabledAdministrativeMonitors.contains(id);
```

Comment:
Why the cast?

Reply:
Because `disabledAdministrativeMonitors` is a package-scoped object. It is not visible from the parent class in another package. Weird, but I do not want to refactor it right now